src/queryClient.ts

```diff
@@ -0,0 +1,13 @@
+import { QueryClient } from 'react-query';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 30000,
```

Comment:
I would suggest to use [numeric separators](https://github.com/tc39/proposal-numeric-separator) for big numbers.

```ts
staleTime: 30_000,
```

Reply:
What do you think of? `staleTime: 30 * 1000, // 30 seconds`

Reply:
That's fine too